README.md

```diff
@@ -37,6 +37,15 @@ Console](https://console.cloud.google.com/apis/credentials/consent).
 To provide these settings, create a file called `settings.json` using the
 instructions in `settings.json-example`.
 
+## Choosing the calendar for events
+
+The example settings file creates calendar events on the primary calendar.
+Instead, a custom name can be specified and the named calendar will be created
```

Comment:
Let's mention that it only creates the named calendar if not present.  So the application will try to first find the calendar with that name, and failing that, will create a new calendar with that name.

Reply:
Done, added 'if it doesn't exist'